permissions/plugin-junit.yml

```diff
@@ -7,6 +7,7 @@ issues:
 paths:
 - "org/jenkins-ci/plugins/junit"
 developers:
+- "@cloudbees-developers"
```

Comment:
Reverting as per Oleg's comment, this way we can move and land the rest and keep discussing the more debatable plugins in a separate PR.

```suggestion
```